test/test_regex.py

```diff
@@ -64,15 +90,25 @@ def test_stripped_jump(self):
         pattern = asm2cfg.get_jump_pattern(True, 'does_not_matter')
         jump_match = pattern.search(line)
 
-        self.assertNotEqual(jump_match, None)
+        self.assertIsNot(jump_match, None)
         self.assertEqual(jump_match[1], '55555555600f')
         self.assertEqual(jump_match[2], '55555555603d')
 
+    @unittest.expectedFailure
+    def test_objdump(self):
+        line = '1017:	eb 06                	jmp    101f <bar+0x1f>'
+        pattern = asm2cfg.get_jump_pattern(False, 'does_not_matter')
+        jump_match = pattern.search(line)
+
+        self.assertIsNot(jump_match, None)
+        self.assertEqual(jump_match[1], '1017')
+        self.assertEqual(jump_match[2], '101f <bar+0x1f>')
```

Comment:
Should the second match only be the jump destination as a hex? In this case it seems to be `101f`, and the omit `<bar+0x1f>, right?

Reply:
> Do you think it would make sense to place unsupported calls as a comments in the file that generate calls to note that these cases are still missing? To have them down somewhere what is still missing.

+1

> Or better yet would be to create an issue of them as well after this is merged.

I'd love to create an issue but it's not yet clear to me how to resolve it...

Reply:
I think if you cannot solve it in this PR then just create an issue which describes what call patterns are missing and where to add them :) When you or somebody will work on the issue they part of that work is to figure out how to generate such calls.

Reply:
Ok. BTW do you remember which executables you used for disasms in `examples/` folder? Was it on Ubuntu?

Reply:
I think the huge one was main function of LLVM frontend clang main function. One of the examples was OpenCL kernel compiled with Intel OpenCL implementation and then GDB dumped. I could look into to this and try to remember.

Reply:
I was thinking would it be possible to generate the calls with registers using somekind of function table and then have a pointer to that table. This would maybe mean that assembly has to keep the pointer to the table in the register and then call it if we increment the pointer to the next function on the that table. Could this help?

Reply:
> I could look into to this and try to remember.

This might be useful. Only open-source code is of use because I'll need to examine their compile/link flags.

> I was thinking would it be possible to generate the calls with registers using somekind of function table and then have a pointer to that table.

There are two options here - function pointers as you mentioned (note that we'll generally have to treat them conservatively i.e. all functions would be reachable from them) and jumptables. I'll try to add template code for both.